### ISSUE TITLE
Impossible to download theme

### ISSUE THREAD

**Issue by AgentleMaap** [RocketRobz/TWiLightMenu-Updater#108]
When i try to download theme , twilight menu close and go back to the main 3ds menu

**Comment by Epicpkmn11** [comment]
Do you have the latest version? (v4.3.0)

**Comment by AgentleMaap** [comment]
Yeah i do the full update of everything and still same trouble...

**Comment by AgentleMaap** [comment]
after reupdate the updtater (without trouble unlike the theme) and still try to download theme , the trouble is still here.
For some really little details that i was able to see:

-after choosing my theme and start the download, a little little graphic glitch appear on the low right corner of the top screen like half of a square, and then i get back to the main menu

-Now some themes i downloaded before become unavailable in TWmenu , but they still appear in the sd card. As a solution i delete the folder and then copy it from an old backup of my sd card.

-when trying to download theme, it seems that a folder is created but is empty on the sd card.

i never get any kind of trouble with TWmenu, it's why this surprise me. I tried to download all the themes and the problem is the same for all.

I have a N3DS XL, i changed recently my 64gb Sd card for 128 gb but i can't rely this change to the problem because when i enter my old 64 gb , i can't download too.
